lib/llvm-backend/src/backend.rs

```diff
@@ -231,10 +238,17 @@ pub struct LLVMBackend {
     module: *mut LLVMModule,
     #[allow(dead_code)]
     buffer: Arc<Buffer>,
+    msm: Option<ModuleStateMap>,
```

Comment:
Style suggestion: I think a more descriptive name should be used for fields: `module_state_map`.

Reply:
I think using `MSM = ModuleStateMap` and `FSM = FunctionStateMap` is fine if we keep it consistent everywhere, since it's quite common and `module_state_map` is too long to type...